### ISSUE TITLE
[TOTK] Tears of the Kingdom stutter/slowdown during first cutscene with Rauru

### ISSUE THREAD

**Issue by Mikhailzrick** [yuzu-emu/yuzu#12064]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Yuzu mainline 1624

### Description of Issue

FPS stable until first cutscene with Raura where during the cutscene fps will slowdown and stutter, then seemingly return to full speed during different parts of the cutscene. On linux so tried using MangoHUD to monitor things and I noticed during the slowdown/stutter GPU usage jumps to 2-3x normal usage(often maxing out and sometimes causing a crash), then returning to normal. It seems consistent as it always happens with this cutscene, but does little to explain why the roller coaster GPU usage during the same cutscene.

Disabling fast gpu time does seem to help as it won't sppike to 99% GPU usage sometimes causing a crash, instead it'll spike to around 80-90% for a few seconds before returning to normal/stable fps. Also enabling handheld mode completely resolves issue 100%.

### Expected Behavior

No slowdown / stuttering during cutscene

### Reproduction Steps

Play beginning part of the game up until the first cutscene with Rauru.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/13398450/yuzu_log.txt)

### System Configuration

CPU: 5700G (oc)
GPU/Driver: 5700G/MESA (oc)
RAM: 32GB (DDR4 4133)
OS: Linux

**Comment by Squall-Leonhart** [comment]
crashes correlating to utilisation implies hardware instability, a slowdown is the expected result of total resource saturation, not a device loss.

**Comment by Mikhailzrick** [comment]
> crashes correlating to utilisation implies hardware instability, a slowdown is the expected result of total resource saturation, not a device loss.

Yes, but does little to explain why there’s such a drastic fluctuation in gpu utilization during the same cutscene, even during the same cut within the same cutscene. With fps unlocked it’ll go from a buttery 45-50 fps to 2 fps then back to a buttery 45-50fps. During gameplay in the over world it will also remain around the 45-50fps range with fps unlocked.

**Comment by Squall-Leonhart** [comment]
The first cutscene is heavy on DOF and this integrated GPU will see util spikes when fetching scene data from the system shared pool, as the 5700G doesn't have dedicated memory.

The crash would be a device loss triggered due to any number of reasons,  in this case its likely one of the  several known memory related cases plaguing pre-RDNA polaris and vega gpu's.

**Comment by Mikhailzrick** [comment]
Interesting. Does that then mean if DOF is the culprit, as you describe, then during gameplay it’s not used? Or at least not to the same degree as during the cutscene? I may see if there’s an available mod to disable DOF entirely to test if it resolves the issue or not. This would be the first time running into a problem directly related to the igpu performance/limitation as using several different benchmarks it’s on par with a gtx 1050/1050 Ti
